manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "manifest_version": 2,
+    "name": "LiveAgent-remote-actions",
+    "version": "1.0",
+
+    "description": "AAAAAAAA",
```

Comment:
This is not very nice description.

Reply:
I'm not really good at describing things. 
![image](https://user-images.githubusercontent.com/8005463/109810568-fc1a0880-7c29-11eb-827d-8106b6c5255e.png)